src/components/CommonSelect/index.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events */
+import classNames from 'classnames'
+import { useState } from 'react'
+import OutsideClickHandler from 'react-outside-click-handler'
+import ArrowDownIcon from '../../assets/arrow_down.png'
+import ArrowDownIconTestnet from '../../assets/arrow_down_blue.png'
+import ArrowUpIcon from '../../assets/arrow_up.png'
+import ArrowUpIconTestnet from '../../assets/arrow_up_blue.png'
+import styles from './index.module.scss'
+import { isMainnet } from '../../utils/chain'
+
+type OptionType = {
+  value: string
+  label: string
+}
+
+type Props = {
+  options: OptionType[]
+  onChange: (value: string) => void
+  defaultValue?: string
+  placeholder?: string
+  className?: string
+}
+
+export default (props: Props) => {
+  const { options, onChange, defaultValue, placeholder, className } = props
+  const defaultLabel = options.find(option => option.value === defaultValue)?.label
+  const [value, setValue] = useState(defaultLabel)
+  const [isExpanded, setIsExpanded] = useState(false)
+  const icons = isMainnet()
+    ? {
+        arrowUp: ArrowUpIcon,
+        arrowDown: ArrowDownIcon,
+      }
+    : {
+        arrowUp: ArrowUpIconTestnet,
+        arrowDown: ArrowDownIconTestnet,
+      }
+  const arrowIcon = isExpanded ? icons.arrowUp : icons.arrowDown
```

Comment:
The direction of arrow can be set by CSS, which is more flexible and animation could be adopted for transition(in the future).

Reply:
updated by [33424cf](https://github.com/Magickbase/ckb-explorer-frontend/pull/127/commits/33424cf29eefcddf2764ed9027af081e6b81bbea)